nbqa/save_source.py

```diff
@@ -305,7 +285,7 @@ def _should_ignore_code_cell(
     ----------
     source
         Source from the notebook cell
-    ignore_cells
+    process_cells
         Extra cells which nbqa should ignore.
```

Comment:
`Extra cells which nbqa should ignore.` should be updated to `Extra cell magics that nbqa should process.`